vshard_shadow_test.go

```diff
@@ -2,28 +2,49 @@ package vshard_router_test
 
 import (
 	"context"
+	"fmt"
 	"testing"
 
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 
-	vshard_router "github.com/KaymeKaydex/go-vshard-router"
+	vshardrouter "github.com/KaymeKaydex/go-vshard-router"
 	"github.com/KaymeKaydex/go-vshard-router/providers/static"
 )
 
+// Check that ErrorTopologyProvider implements TopologyProvider interface
+var _ vshardrouter.TopologyProvider = (*ErrorTopologyProvider)(nil)
+
+type ErrorTopologyProvider struct{}
```

Comment:
No need to expose this type `ErrorTopologyProvider` -> `errorTopologyProvider`

Reply:
done